src/tests/JIT/Regression/JitBlue/Runtime_35144/Runtime_35144.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <CLRTestPriority>1</CLRTestPriority>
```

Comment:
Sorry if I totally miss the ball here, but wasn't **Runtime_35724** supposed to be marked as prio 1 ?

Reply:
yes, it was fixed in https://github.com/dotnet/runtime/pull/39062